# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

GraspFActionGoal action_goal
GraspFActionResult action_result
GraspFActionFeedback action_feedback
